rules/order/tests/StmtOrderTest.php

```diff
@@ -66,6 +66,6 @@ public function testReorderClassStmtsByOldToNewKeys(): void
         $expectedClass->stmts[] = new Property(Class_::MODIFIER_PRIVATE, [new PropertyProperty('name')]);
         $expectedClass->stmts[] = new Property(Class_::MODIFIER_PRIVATE, [new PropertyProperty('price')]);
         $expectedClass->stmts[] = new Property(Class_::MODIFIER_PRIVATE, [new PropertyProperty('service')]);
-        $this->assertEquals($expectedClass->stmts, $actualClass->stmts);
+        $this->assertSame($expectedClass->stmts, $actualClass->stmts);
```

Comment:
@TomasVotruba what if I'd like to use assertEquals here? :)

Reply:
I actually wonder that we have such an issue https://github.com/rectorphp/rector/issues/3742 as it looks like there is a rule which replaces assertEquals to assertSame

Reply:
> it looks like there is a rule

Where is it?

Reply:
I never know which is which :D we use the strict one. It should work here too

Reply:
> > it looks like there is a rule
> 
> Where is it?

I've just checked and didn't find any... I wanted to use `assertEquals` because I'd like to compare two different instances of the object but CI changed it to `assertSame`

Reply:
> I never know which is which :D we use the strict one. It should work here too

I see but in this particular case `assertEquals` suites better. In case we are so strict I'll have to create more asserts and to compare method names. Also, it seems that it is not that easy to resolve the method names in this kind of tests.

Reply:
I see, the new nodes are created manually and thus are different.

Reply:
Check it out https://github.com/rectorphp/rector/pull/3884/commits/12df4706265f126a21552cb2d056a27adeb6f087